BlazarBase/src/main/java/com/hubspot/blazar/base/BuildTrigger.java

```diff
@@ -31,6 +31,11 @@ public static BuildTrigger forBranchCreation(String branch) {
     return new BuildTrigger(Type.BRANCH_CREATION, branch);
   }
 
+  public static BuildTrigger forInterProjectBuild(GitInfo gitInfo) {
+    String rootRepoBranch = String.format("%s-%s-%s-%s", gitInfo.getHost(), gitInfo.getOrganization(), gitInfo.getRepository(), gitInfo.getBranch());
+    return new BuildTrigger(Type.INTER_PROJECT, rootRepoBranch);
```

Comment:
Can we put the `id` of the row in the interproject build table? Or does that create a chicken and egg problem with the way things are currently structured?